cmd/vicadmin/server.go

```diff
@@ -314,16 +323,15 @@ func (s *server) loginPage(res http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	// Render login page (shows up on non-POST requests):
-	sess, err := client(&rootConfig)
-	if err != nil {
-		log.Errorf("Could not render login page due to vSphere connection error: %s", err.Error())
-		http.Error(res, genericErrorMessage, http.StatusInternalServerError)
-		return
+	// Render login page (shows up on non-POST requests)
+	hostName, _ := os.Hostname()
```

Comment:
If the error is non-nil, `hostName` would be an empty string, which is what we'd want to display if we can't find the hostname. Please let me know if you have suggestions.